config/sys.config.src

```diff
@@ -1,5 +1,10 @@
 [
     {hpr, [
+        {aws_config, [
+            {access_key_id, <<"${AWS_ACCESS_KEY_ID}">>},
```

Comment:
Do you want binaries here? It seems like in other config you are using strings

Reply:
Yes, they should be binaries. Fixed to make types consistent.